pkg/flexvol/flexvol.go

```diff
@@ -432,6 +396,9 @@ func retryGetDockerPathAndMetadata(flexvolPath, devPath string) (string, string,
 			}
 			return dockerPath, metadata, err
 		}
+		if err != nil {
```

Comment:
unreachable code..did it compile for you?

Reply:
Yes this has compiled and possiblly due to multiple merge it cropped in . thx for catching.